src/map/Map.js

```diff
@@ -71,11 +73,13 @@ L.Map = L.Evented.extend({
 	},
 
 	zoomIn: function (delta, options) {
-		return this.setZoom(this._zoom + (delta || 1), options);
+		delta = delta || (L.Browser.any3d ? this.options.zoomDelta : 1);

```

Comment:
There are a few checks for `L.Browser.any3d` scattered around. This is a bit dirty, but I don't think it's better to have `map.setZoomDelta()` and `map.setZoomSnap()` functions to account for old browsers with no CSS transforms.